lnrpc/rpc.proto

```diff
@@ -2516,6 +2527,25 @@ message ChannelGraph {
     repeated ChannelEdge edges = 2;
 }
 
+message NodeMetricsRequest {
```

Comment:
imo the request should accept an (repeated) enum stating which metric(s) we want to collect. otherwise we will always need to compute betweenness centrality even if we add more metrics because we won't have a way to stop computing betweenness centrality in a backwards compatible way.

Reply:
Agree PTAL